### ISSUE TITLE
Item/Event/Location names look weird?

### ISSUE THREAD

**Issue by Razzmatazzz** [Razzmatazzz/RemnantSaveGuardian#45]
"Translating" the names for items, events, and locations from their names in the save files to what you see in the game is an ongoing process. If you recognize that an particular event has a strange name in the analyzer but is called something totally different in-game and/or on the wiki, please post it here.

**Comment by sinnoaria** [comment]
![image](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/12928534/614794ef-08b7-49bd-bc10-46f4e5c43a7e)
Not sure if it belongs here, but highlights are hiding the name of the items and custodian is duplicated with one in phantom wasteland and one in generic nerud.

Also, Nerud: Towers_Test shows up for The Astropath. and Nerud: Underground_Infection shows up where the Inert Overcharger is. Nerud: Underground_Stasis for Abomination.

**Comment by Razzmatazzz** [comment]
What other events show up in Underground_Infection and Underground_Stasis?

**Comment by sinnoaria** [comment]
> What other events show up in Underground_Infection and Underground_Stasis?  I can't remember for underground infection, but I have underground stasis a few times.
One world:
![image](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/12928534/1b6a5662-9d39-4d11-842c-1aca9e29c97d)
Another world:
![image](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/12928534/1f11063f-c65f-4c2e-873f-3647a6e9c86c)
Another world:
![image](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/12928534/ef49d029-f4e1-45eb-8150-c3cec6f8b91c)
And one last world:
![image](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/12928534/7bbb9efe-0dcf-4cdb-aedd-b15bf505cc43)

**Comment by Razzmatazzz** [comment]
These will all be fixed in the next version.

**Comment by sinnoaria** [comment]
Vault of the formless in .10 is listed as VaultOfTheFormless. Not sure if that was fixed in .11

**Comment by sinnoaria** [comment]
![image](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/12928534/1eec6010-c1d2-429f-84e8-62e79e788fd8)

**Comment by Dom-1** [comment]
Shotgun should (probably?) be renamed "Ford's Scattergun". Right now it leads to 404 when looking for it on the wiki.
https://remnant2.wiki.fextralife.com/Ford's+Scattergun

**Comment by Razzmatazzz** [comment]
It might also be the coach gun. Need to know what each is called in the game files.

**Comment by Dom-1** [comment]
I am specifically talking about the "Shotgun" event at the Red Throne.
That is definately "Ford's Scattergun".

**Comment by crackedmind** [comment]
> Shotgun should (probably?) be renamed "Ford's Scattergun". Right now it leads to 404 when looking for it on the wiki. https://remnant2.wiki.fextralife.com/Ford's+Scattergun

Check game resources, and you're right.

> I am specifically talking about the "Shotgun" entry at the Red Throne in Yaesha. That is definately "Ford's Scattergun".
> 
> Edit: The armor set that is called "Armor_Arcanist (XY)" in the program is called Labyrinth in the game. https://remnant2.wiki.fextralife.com/Labyrinth+Set
> 
> Edit2: Ring_BandOfAccord should probably be renamed and link to the correct page on the wiki. https://remnant2.wiki.fextralife.com/Band+of+Accord

I updated this here https://github.com/Razzmatazzz/RemnantSaveGuardian/pull/86

**Comment by AndrewSav** [comment]
Invader armor as reported by RemnantSaveGuardian is called Dendroid armor in-game.

**Comment by crackedmind** [comment]
> Invader armor as reported by RemnantSaveGuardian is called Dendroid armor in-game.

Updated in PR. Waiting for merge.

**Comment by AndrewSav** [comment]
![2023-08-15_23-33-29](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/658865/0bf3f01d-82cb-4035-a014-67583916ee9a)

So all these are in The Withering Weald, not in the Far Woods. The Withering Weald in this adventure save is the first overworld zone and it has the Flautist, The Far Woods in the second overworld zone, and these events are not in it.

[data.zip](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/12343705/data.zip)

save_1 (second char) adventure mode.

**Comment by Razzmatazzz** [comment]
I initially used the method of pulling location names directly from the save files, but it wasn't correct in some instances. I'm not too inclined to go back to it given the amount of reports of inaccuracies it generated.

**Comment by AndrewSav** [comment]
@Razzmatazzz I think it would be possible to find middle ground. If you are sure that the location is fixed, then fine, but in case such as I reported the location can change which cause it to be wrong which causes the inaccuracy. Surely changing it **for those cases** cannot make it worse?

**Comment by Razzmatazzz** [comment]
I'm not saying the location is always correct. It isn't. I'm saying pulling the location name text directly out of the save files can also be inaccurate and generate reports of inaccuracies. May be possible to use manual location names when there's a known 1:1 association between events and locations and then use the text from the save file for instances where an event can occur in multiple locations.

**Comment by GitBooSTid** [comment]
![ChimneyApp](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/40153438/58f9da10-7750-44e3-8579-e52e00db9e66)
"Yaesha: Laemirs' Lyceum" should be "Yaesha: The Chimney"

It's also missing the ring "Berserker's Crest"
[Berserker's Crest - Wiki](https://remnant2.wiki.fextralife.com/Berserker's+Crest)
![Chimney](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/40153438/51b801f3-6b58-4c73-9773-d2f5310dc814)

**Comment by Razzmatazzz** [comment]
@GitBooSTid can you please post your save (profile and world)? I want to see if there's an event for the coffin.

**Comment by GitBooSTid** [comment]
@Razzmatazzz In case you still want it for whatever reason -- [Save.zip](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/12383655/Save.zip)

**Comment by Razzmatazzz** [comment]
@GitBooSTid and this save is from the same campaign roll as above? It's strange because I'm seeing completely different events in The Chimney.

**Comment by GitBooSTid** [comment]
Yes, but in the above, I was showing you a screenshot from the adventure tab, not campaign. I have The Chimney in both campaign and adventure at the moment.

**Comment by Razzmatazzz** [comment]
Ah, got it. This should be working right in the latest release.

**Comment by AndrewSav** [comment]
> This build attempts to combine both methods of determining locations:
> 
> [RemnantSaveGuardian.zip](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/12371200/RemnantSaveGuardian.zip)

This build still displays scavenger babble in Far Wood where as it is in Withering Weald. The latest build 1.0.2.2 displays scavenger babble in the Nameless nest which is a side dungeon.

**Comment by AndrewSav** [comment]
> This build attempts to combine both methods of determining locations:
> 
> [RemnantSaveGuardian.zip](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/12371200/RemnantSaveGuardian.zip)

Sorry, was not playing a last few days. So yes, this builds displays this item in Withering Weald and the latest (1.0.2.2) displays them in Root Nexus, which is also in Withering Weald so it works for me. Thank you.

**Comment by AndrewSav** [comment]
However, in the last build, Spirit Stone is displayed in The Far woods, _the experimental build above showed it correctly in Kaeula's Rest_ (same save, adventure mode) ![2023-08-20_01-19-00](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/658865/bf1c6dfa-b3bb-4d32-b91d-fc2e5445b4bb) 
![2023-08-20_01-20-15](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/658865/c72befa3-c281-4d55-a68b-1c4f22914276)
![2023-08-20_01-26-16](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/658865/7fba6113-0ee0-4173-8ae0-0f62dbc1ede3)

Same method I suggested last time would work here.

**Comment by Razzmatazzz** [comment]
How it works currently is if the first even in an area has been assigned a location manually, then all events in that area use the manual location. If a location has not been assigned, then it uses the area name present in the save file.

**Comment by GitBooSTid** [comment]
![GreatSewers](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/40153438/5790bd5e-6343-40dd-bf94-103efb15e11f)

The above event/injectable is listed in The Great Sewers, but is only an overworld tile event. For me, this occurred in Ironborough. There is the same event in the underground/sewers tilesets, which I also have in my campaign. This is the same save as I uploaded in this thread previously.

**Comment by AndrewSav** [comment]
@Razzmatazzz since the experimental build seemed to work fine and the latest one does not may be it's best to do what the experimental one does? I do understand that it's a subtle balancing act and what fixes something breaks other things, but I'm not aware of what the experimental might have broken.

**Comment by GitBooSTid** [comment]
![Ribbon](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/40153438/5aea7d48-a746-4d3f-8988-7ef10476a816)

Not sure if this is one you care about, but you can't get Amulets: Silver Ribbon if "The Statues" appears on Gilded Chambers or Council Chambers tilesets, only if it's in Shattered Gallery or The Great Hall. The inverse is true for the Golden Ribbon.

[Plain Ribbon](https://remnant2.wiki.fextralife.com/Plain+Ribbon)

**Comment by aescolastico** [comment]
@GitBooSTid The monster does spawn in the sewers as well. but when it does, it doesnt reward the alchemist crafting material. just the spawns you in the area with the alchemist armor set.

**Comment by GitBooSTid** [comment]
@aescolastico I understand and I even described that in my original post. I assume we're trying to clean this up and only list what is possible... which in this case it is not possible to get "Engrams: Alchemist" in Losomn: The Great Sewers, which is what the app is stating.

**Comment by aescolastico** [comment]
@GitBooSTid You misunderstand. I was adding context to your report. I wasnt saying you were wrong. What you showed indeed does not reflect the items location in game. We are in agreement.

**Comment by Razzmatazzz** [comment]
This is related to an issue wherein injectable events do not always appear in the area they should in the save file. The analyzer attempts to insert them where they should be, but it isn't always possible to accurately insert them.

**Comment by DanMartin85** [comment]
not sure if this is the right spot. but the wiki link for Havigator's Helm goes to a 404 page.

correct link should be: https://remnant2.wiki.fextralife.com/Navigator's+Helm

**Comment by SveItE** [comment]
Location called "Losomn: Asylum: Asylum: Tormented Asylum" probably doesn't need to be called all that. 

![Remnant2-Win64-Shipping_9KSBH2IAni](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/142968910/7064bd73-259b-46e4-8bd8-0984874aa078)

**Comment by Razzmatazzz** [comment]
> > not sure if this is the right spot. but the wiki link for Havigator's Helm goes to a 404 page.
> > correct link should be: https://remnant2.wiki.fextralife.com/Navigator's+Helm
> 
> Yep, also for [Lodestone Crown](https://remnant2.wiki.fextralife.com/Lodestone+Crown) and [Crown of the Red Prince](https://remnant2.wiki.fextralife.com/Crown+of+the+Red+Prince) because " Set" added

The wiki links for these should be fixed in the latest release.

**Comment by Razzmatazzz** [comment]
> Location called "Losomn: Asylum: Asylum: Tormented Asylum" probably doesn't need to be called all that.

Yeah, it's redundant, but it also follows the convention for other events of World: Area Name: Event Name. It might be better if it was just Losomn: Asylmy or Losomn: Tormented Asylum, but ¯\_(ツ)_/¯

**Comment by Eenna** [comment]
Hi! Read through the thread, didn't see this specific duplicated example: 
![Image](https://github.com/user-attachments/assets/0e223e61-9d2a-48bc-9fb5-fd6742ca28ca)
A comment from 2023 said duplication is an ongoing issue, but I'm not sure if there's been updates in that regard. Thanks!
